macros/sql/fetch.sql

```diff
@@ -0,0 +1,21 @@
+{% macro fetch(sql) %}
+{# This macro returns a dictionary of the form {column_name: [results]} #}
+
+    {%- call statement('get_sql_results', fetch_result=True) -%}
+
+        {{sql}}
+
+    {%- endcall -%}
+    
+    {% set sql_results={} %}
+    
+    {%- if execute -%}
+        {% set sql_results_table = load_result('get_sql_results').table.columns %}
```

Comment:
We can probably just return the table here, right? What's the rationale behind pushing these values into a dict?

Reply:
That was my rationale – to operate on the Table, you need to know about agate's [Table object](https://agate.readthedocs.io/en/1.6.1/api/table.html), which is one more set of docs to go look up

I can see someone finding `fetch` to be generally useful separate from `get_column_from_sql`, so wanted to return an object that was familiar to a user (i.e. a dictionary).

I am happy either way though!

Reply:
Yes +1 as a pythonista I want a dict pl0x